controllers/nodefeaturediscovery_resources.go

```diff
@@ -58,8 +62,17 @@ func Add3dpartyResourcesToScheme(scheme *runtime.Scheme) error {
 	return nil
 }
 
+// filePathWalkDir takes a path as an input and finds all files
+// in that path, but not directories
```

Comment:
Again, an editorial comment for streamlining the comment 😄  

WDYT?

```suggestion
// filePathWalkDir finds all non-directory files under the given path recursively,
// i.e. including its subdirectories
```